### PR TITLE
Fix/validate context

### PR DESCRIPTION
**Pull request by FredrikOseberg**

### CODE REVIEW

**Review by sighphyre — APPROVED** [review]
LGTM

File: src/lib/services/feature-toggle-service.ts [hunk 1]
```diff
@@ -569,10 +572,11 @@ class FeatureToggleService {
...
         const preData = await this.featureToggleStore.get(featureName);
 
-        const featureToggle = await this.featureToggleStore.update(
-            projectId,
-            featureData,
-        );
+        const featureToggle = await this.featureToggleStore.update(projectId, {
+            ...featureData,
+            name: featureName,
+        });
```
Comment (ivarconr) [inline src/lib/services/feature-toggle-service.ts:578]: This works 👍🏼 

In general terms it is quite common pattern that you have to specify the id of the thing you are updating, and we have already taken projectId out of the metadata paylaod. Would it make sense that the update also takes featureName as a separate argument and that we never actually change the name of the feature toggle (this is nothing our API supports anyway. The only option you have as a user is to archive/delete and create a new toggle). 

what do you think?

**Review by ivarconr — APPROVED** [review]

File: src/lib/services/feature-toggle-service.ts (hunk 1 above)
Comment (FredrikOseberg) [reply src/lib/services/feature-toggle-service.ts:578]: This works as well, and could well be a better solution. The only thing we'd need to be mindful of is that we'll need to adjust the joi schema to send 400 bad request if the payload includes name (or remove the name from the payload once we receive it).

Comment (FredrikOseberg) [reply src/lib/services/feature-toggle-service.ts:578]: I'll merge this and put a note in the backlog on splitting out the featureName as a separate argument to the store method.

**Pull request merged**